dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -80,3 +80,12 @@ preferCRLs.desc=By default, OCSP is the preferred mechanism for checking revocat
 
 disableFallback=No fallback
 disableFallback.desc=Disable the certificate revocation status checker fallback mechanism. During certificate revocation status check, the default behavior is fallback and check by using the non-preferred mechanism (OCSP or CRLs). Default is false.
+
+certCheckerSchedule=Certificate checker schedule
+certCheckerSchedule.desc=Interval for the ACME CA service to check if the certificate is expiring or revoked. If the certificate is expiring or revoked, the ACME CA service issues a certificate renew request. If the first renew request fails, the certificate renew request continues according to the timing set by the certCheckerErrorSchedule attribute until a new certificate is received. Setting the certCheckerSchedule property to zero or a negative value disables automatic certificate renew requests.
+
+certCheckerErrorSchedule=Certificate checker error schedule
+certCheckerErrorSchedule.desc=Performs the same function as the certCheckerSchedule attribute, but on an alternate schedule. For example, the certCheckerErrorSchedule attribute can be set to a shorter interval than the certCheckerSchedule attribute. The interval from the certCheckerSchedule attribute is resumed after the certificate is renewed. 
```

Comment:
For example, the certCheckerErrorSchedule attribute can be set to a shorter interval than the certCheckerSchedule attribute. 
 
This is better, but can we say why someone would want to do that? like:

 For example, the certCheckerErrorSchedule attribute can be set to a shorter interval than the certCheckerSchedule attribute, to increase the frequency of checks after a failed request.

Reply:
Added